test/setup-database.js

```diff
@@ -81,3 +136,9 @@ exports.ConnectionFactory = () =>
     endpoint: 'http://localhost:5820',
     // endpoint: 'http://localhost:61941',
   });
+
+exports.icvAxioms = () =>
```

Comment:
Why do you want this to be a function? You can just make it the result of the read operation.